fk/secretreceive.go

```diff
@@ -84,10 +84,15 @@ func secretReceive() exitCode {
 
 		for _, secret := range decryptedSecrets {
 			out.Print(formatSecretListItem(secret.decryptedContent))
+			if prompter.promptYesNo("Delete now?", "Y", nil) == true {
+				err := client.DeleteSecret(secret.sentToFingerprint, secret.UUID.String())
```

Comment:
seems we no longer need `secret.sentToFingerprint` if we're deleting at this point... right?